rust/src/cli/ncl.rs

```diff
@@ -138,7 +139,7 @@ fn main() {
         .subcommand(
             clap::Command::new(SUB_CMD_APPLY)
                 .about("Apply network state or network policy")
-                .alias("set")
```

Comment:
Please do not remove existing aliases, people might be using them and they will suddenly stop working.

Reply:
thanks so much @ihuguet reverting that

Reply:
any pointer about the failed checks @ihuguet 
one seem to be related to fedora

Reply:
Also I suppose we need to update the documents for users to know of the new aliases

Reply:
I think you can ignore the Fedora failure, it seems a dependency issue to a package that is not updated in Fedora yet. Certainly not related to your changes.

About the documentation: good point! Honestly I don't know if we have any of the other alias documented.